src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/ResourceGroupDeployments/NewAzureResourceGroupDeploymentCmdlet.cs

```diff
@@ -45,6 +45,9 @@ public class NewAzureResourceGroupDeploymentCmdlet : ResourceWithParameterCmdlet
         [ValidateSet("RequestContent", "ResponseContent", "All", "None", IgnoreCase = true)]
         public string DeploymentDebugLogLevel { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of a deployment to roll back to on error, or use as a flag to roll back to the last successful deployment.")]
```

Comment:
This needs to have an argument completer that allows the user to choose 'LastSuccessfulDeployment'  - A user selecting this is the way you should choose to send 'LastSuccessful' in the code below